colibre/scripts/density_species_interp.py

```diff
@@ -0,0 +1,339 @@
+"""
+Makes a rho-T plot. Uses the swiftsimio library.
```

Comment:
Please add a longer decsription of the specific function of this rho-T instance.

Reply:
changed